src/app/screens/confirmOrdinalTransaction/index.tsx

```diff
@@ -53,12 +52,24 @@ function ConfirmOrdinalTransaction() {
     fee = BigNumber(fee);
   }
   const { id } = useParams();
-  const { data: selectedOrdinal } = useAddressInscription(id!);
-  const { setSelectedSatBundleDetails } = useSatBundleDataReducer();
+  const [txid, vout] = id!.split(':');
+  const { data: selectedOrdinal } = useAddressInscription(txid!);
   const { refetch } = useBtcWalletData();
   const [currentFee, setCurrentFee] = useState(fee);
   const [currentFeeRate, setCurrentFeeRate] = useState(feePerVByte);
 
+  const {
+    bundle: ordinalBundle,
+    isPartOfABundle,
+    ordinalSatributes,
+  } = useGetUtxoOrdinalBundle(
+    selectedOrdinal?.output || id,
+    hasActivatedRareSatsKey,
+    selectedOrdinal?.number,
+  );
+
+  const holdsRareSats = ordinalSatributes?.length > 0;
+
```

Comment:
I wonder if there's a cleaner way for this route to be able to handle both the send ordinal flow and send rare sat flow